src/systems/physics/Physics.cc

```diff
@@ -212,6 +212,11 @@ class ignition::gazebo::systems::PhysicsPrivate
   /// \brief Keep track of what entities are static (models and links).
   public: std::unordered_set<Entity> staticEntities;
 
+  /// \brief Keep track of non-static link poses.
+  /// This allows for skipping pose updates if a link's pose didn't change
+  /// after a physics step.
```

Comment:
In a model, there will be links that never move over time (i.e. ones not connected by joints). Looking at the code below, does `non-static link poses` here refer to its parent not being static?

Reply:
> Looking at the code below, does non-static link poses here refer to its parent not being static?

Yes, that is the correct interpretation. I've updated the documentation in https://github.com/ignitionrobotics/ign-gazebo/pull/669/commits/c88b4747c9a1b1d6f77fb61a4e797d417a71d195 to make this clear. If it looks good to you, we can mark this as resolved.

Reply:
looks good to me.